lms/static/certificates/sass/_config.scss

```diff
@@ -0,0 +1,37 @@
+// ------------------------------
+// Open edX Certificates: Config
+
+// About: variable and configuration overrides
+
+// #VARIABLES
+
+// ------------------------------
+// #VARIABLES
+// ------------------------------
+$em-base:       16; // deliberately sets bourbon-based em-base (http://bourbon.io/docs/#em-base)
```

Comment:
Sure thing!

``$em-base`` is a [configuration variable in the UXPL](https://github.com/edx/ux-pattern-library/blob/master/pattern-library%2Fsass%2Fglobal%2F_settings.scss#L27) (that this app is overriding/customizing it). This configuration is used in any of our (Bourbon-based) rem-to-px Sass utilities (see our [font-size function](https://github.com/edx/ux-pattern-library/blob/master/pattern-library%2Fsass%2Fglobal%2F_functions.scss#L20)).

Since the value is the same as the default value used in the UXPL, this line isn't needed. But if an app ever wanted to set its own different base (other than the current, ~16px), this is how to do it per app Sass/CSS. I'll remove this line since its redundant for this case.